.packit.yml

```diff
@@ -0,0 +1,21 @@
+specfile_path: .ci/packit/rpm/systemd.spec
+synced_files:
+  - .packit.yaml
+  - src: .ci/packit/rpm/systemd.spec
+    dest: systemd.spec
+upstream_package_name: systemd
+downstream_package_name: systemd
+# git describe returns in systemd's case 'v245-xxx' which breaks RPM version
+# detection (that expects 245-xxxx'. Let's tweak the version string accordingly
+current_version_command: ["bash", "-c", "git describe --tags | grep -Eo '[0-9]+.*$'"]
+actions:
+  post-upstream-clone:
+    - "git clone https://src.fedoraproject.org/rpms/systemd .ci/packit/rpm --depth=1"
+    - "sed -ri '/^Patch[0-9]+\\:.+\\.patch/d' .ci/packit/rpm/systemd.spec"
```

Comment:
Some patches should be kept. Maybe we should split the patch list upstream: use Patch0900+ for patches that are downstream specific and should not be removed. Use Patch0000-0899 for patches that are backported from upstream.

This would resolve the issue with the failing test too.

Reply:
> Some patches should be kept. Maybe we should split the patch list upstream:

you meant *downstream* (Fedora)?

> use Patch0900+ for patches that are downstream specific and should not be removed. Use Patch0000-0899 for patches that are backported from upstream.
> 
> This would resolve the issue with the failing test too.

That would be great, if it's not a too much hassle for you.